components/project-details/challenges/ChallengesList.tsx

```diff
@@ -15,15 +10,16 @@ const ChallengesLearningsList = ({
     </h4>
     <ul>
       {items.map((item, index) => (
-        <ChallengesLearningsItem
+        <ChallengesItem
           key={index}
           text={item}
           title={title}
           iconSrc={iconSrc}
+          index={index}
```

Comment:
I'm sure we could use a more accurate key than the index, right? It doesn't really matter if there's no reordering changing, but try to build your code to be future proof in-case a more dynamic feature is added. Normally I wouldn't correct something like this but you're going very in depth with the animations and features of this project- I wouldn't put it passed you to start implementing dynamically reordering elements ;)

Reply:
Updated Schema and query so each ChallengesItem has a unique key